build-system/babel-plugins/babel-plugin-transform-stringish-literals/index.js

```diff
@@ -34,79 +30,78 @@ module.exports = function ({types: t}) {
     }
   }
 
+  function joinTemplateLiterals(left, right) {
+    // First merge the first member of the right quasi into the last quasi on the left.
+    const fromQuasi = right.get('quasis.0');
+    const toQuasi = left.get(`quasis.${left.node.quasis.length - 1}`);
+    toQuasi.node.value.raw += fromQuasi.node.value.raw;
+    toQuasi.node.value.cooked += fromQuasi.node.value.cooked;
```

Comment:
Nit:

```suggestion
    const fromQuasi = right.node.quasis[0];
    const toQuasi = left.node.quasis[left.node.quasis.length - 1];
    toQuasi.value.raw += fromQuasi.value.raw;
    toQuasi.value.cooked += fromQuasi.value.cooked;
```

Reply:
Unfortunately we need `fromQuasi` to be the path so we can remove it before merging the right quasis.

We could do a filter during the merge instead, but this current approach makes the logic clear with comments.

Reply:
Should be resolved, added a comment about why we need the paths.